### PR TITLE
get raw data in callback notification

### PR DESCRIPTION
**Pull request by nickhargreaves**
$_REQUEST is meant to only get ($_GET or $_POST)

### CODE REVIEW

File: reciver_beyonic_ipn.php [hunk 1]
```diff
@@ -8,20 +8,16 @@
...
-    $hook_array = json_decode($finalHook);
-    $event = $hook_array->event;
-    if ($event == 'collection.recieved') {
-        $order_id = $data_array->metadata->order_id;
-        $state = $data_array->state;
+    $data = $responce->data;
+    $hook = $responce->hook;
+    $event = $hook->event;
+    if ($event == 'collection.received') {
+        $order_id = $data->metadata->order_id;
```
Comment (KayLuke) [inline reciver_beyonic_ipn.php]: I think the order id is on the metadata on the collection request object (not the collection object) so we need to retrieve that first.

Comment (nickhargreaves) [inline reciver_beyonic_ipn.php]: Wasn't sure about that because we are going to include the expanded collection request object in the result

Comment (KayLuke) [inline reciver_beyonic_ipn.php]: Yeah, although that won't be for another couple of weeks

Comment (nickhargreaves) [inline reciver_beyonic_ipn.php]: Updating...

Comment (KayLuke) [inline reciver_beyonic_ipn.php]: Thanks for the latest changes but I see that there is now a conflict? The dev might have edited some files. Please fix and I'll merge.

Also, what happens if there is no collection request for some reason? I can ask the dev to look at this scenario.

Comment (nickhargreaves) [inline reciver_beyonic_ipn.php]: Sure. I've fixed the conflict

**Pull request merged**